source/common/tcp/original_conn_pool.cc

```diff
@@ -38,6 +38,7 @@ OriginalConnPoolImpl::~OriginalConnPoolImpl() {
 }
 
 void OriginalConnPoolImpl::drainConnections() {
```

Comment:
It's unfortunate we still have this code. Are we close to being able to delete it? Wondering if we need to bother making these changes or not.

Reply:
Yes, this needs to live for another few months. @alyssawilk is close to killing it, but we're not there yet. But close.